src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.WinAppSdk.targets

```diff
@@ -14,7 +14,7 @@
 		<_UnoProjectSystemPackageReference Include="SkiaSharp.Views.WinUI" ProjectSystem="true" />
 	</ItemGroup>
 
-	<ItemGroup Condition="$(UnoFeatures.Contains(';canvas3dgl;'))">
+	<ItemGroup Condition="$(UnoFeatures.Contains(';GLCanvas;'))">
```

Comment:
This should be `glcanvas` (lowercase) I think we we normalize UnoFeatures to lower case